.gitignore

```diff
@@ -95,3 +96,4 @@ obsolete/*
 **/credentials/repo_*
 JPS_VIRTUALSENSOR/ShipInputAgent/data/*.json
 *.tif
+/UK_Digital_Twin/UK_Digital_Twin_Package/PYPOWER
```

Comment:
Moved.

Reply:
Rather than moving just that one line, it appears you have deleted the whole file. Please _NEVER_ delete the top-level `.gitignore` file - this is a very important file. Please restore it.

Reply:
Restored.